app/views/programs/show.html.erb

```diff
@@ -1,74 +1,22 @@
-          <% if review.edited? %>
-            <i>- edited
-              <%= time_ago_in_words(review.updated_at) %> ago
-            </i>
-          <% end %>
-          <br>
-          <% if review.user == current_user %>
-            <%= link_to "Edit Comment",
-            edit_program_review_path(@program, review) %><br>
-          <% end %>
-          <% if review.user == current_user || current_user.try(:admin?) %>
-            <%= link_to "Delete Comment",
-             program_review_path(@program, review), method: :delete %>
-          <% end %>
-          <div class="vote-buttons" id="<%= review.id %>">
-            <% if review.votes.where(user: current_user).any? %>
-              <% if review.votes.where(user: current_user).first.value == 1 %>
-                <a class="vote vote-up" id="upvoted">▲</a>
-                <a class="vote vote-down" id="downvote">▼</a>
-              <% elsif review.votes.where(user: current_user).first.value == -1 %>
-                <a class="vote vote-up" id="upvote">▲</a>
-                <a class="vote vote-down" id="downvoted">▼</a>
-              <% else %>
-                <a class="vote vote-up" id="upvote">▲</a>
-                <a class="vote vote-down" id="downvote">▼</a>
-              <% end  %>
-            <% else %>
-              <a class="vote vote-up" id="upvote">▲</a>
-              <a class="vote vote-down" id="downvote">▼</a>
-            <% end %>
-            Review Score: <span class="score"><%= review.get_score %><span>
-          </div>
-        </li>
-      </div>
-    <% end %>
-  </ul>
-<% end %>
-
-<h2>Add a review!</h2>
-<%= form_for [@program, @review] do |f| %>
-  <%= f.label :rating %>
-  <%= f.text_field :rating %>
-
-  <%= f.label :body %>
-  <%= f.text_field :body %>
-
-  <%= f.submit ("Add Review")%>
-<% end %>
-
-<% if current_user.try(:admin?) %>
-  <%= link_to 'Edit', edit_program_path(@program) %><br>
-  <%= link_to 'Delete Program', program_path, method: :delete %>
-<% end %>
+<div class="flex-video widescreen">
+  <iframe width="854" height="480" src="//www.youtube.com/embed/<%= @program.url %>" frameborder="0"></iframe>
+</div>
+
+<div class="medium-9 columns">
+  <h1 class="show-title"><%=@program.title%></h1>
+  <% 5.times do %>
+    <img class="star" src="https://cdn2.iconfinder.com/data/icons/windows-8-metro-style/128/star.png" />
+  <% end %><a href="#" class="button" id="review-button">Add a review</a>
+  <h3> Average rating: <%= @program.reviews.average(:rating) %> </h3>
+  <div class="hide" id="add-review">
+    <%= render "programs/add_review" %>
+  </div>
+    <%= render "programs/reviews" %>
+</div>
+<div class="medium-3 columns">
+    <%= render "layouts/search_bar" %>
+  <h3>Watch Next</h3>
+  <img src="http://img.youtube.com/vi/ftrwe16sBiA/mqdefault.jpg" /> </br ></br >
+    <img src="http://img.youtube.com/vi/ftrwe16sBiA/mqdefault.jpg" /><br /></br >
+      <img src="http://img.youtube.com/vi/ftrwe16sBiA/mqdefault.jpg" />
```

Comment:
Be sure to observe proper indentation. All of these `img` tags should be at the same indentation level.